tests/test_helper.go

```diff
@@ -0,0 +1,212 @@
+package tests
+
+import (
+	"context"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+	"os"
+	"strings"
+)
+
+// common constants
+const (
+	TestObjectName         = "foo"
+	TestNamespace          = "openshift-migration"
+	TestMigrationNamespace = "ocp-41583-longpvcname"
+	TestSecretType         = "Opaque"
+)
+
+// cluster constants
+const (
+	TestSourceCluster      = "source"
+	TestDestinationCluster = "host"
+	TestClusterSecret      = "fooclustersecret"
+	//system variables
+	EXPOSEDREGISTRYPATH = "EXPOSEDREGISTRYPATH"
+	SOURCEURL           = "SOURCEURL"
+	SOURCECONFIG        = "SOURCECONFIG"
+	HOSTCONFIG          = "KUBECONFIG"
+)
+
+// storage constants
+const (
+	TestStorageSecret = "foostoragesecret"
+	ConfigNamespace   = "openshift-config"
+	// system variables
+	AWSBUCKETNAME         = "AWSBUCKETNAME"
+	AWSSECRETKEY          = "AWSSECRETKEY"
+	AWSACCESSKEY          = "AWSACCESSKEY"
+	BACKUPSTORAGEPROVIDER = "BACKUPSTORAGEPROVIDER"
+)
+
+// migrationcontroller constants
+const (
+	MigrationController = "migration-controller"
+)
+
+func NewMigMigration() *migapi.MigMigration {
+	return &migapi.MigMigration{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      TestObjectName,
```

Comment:
the same comment as above, does the name have to be a constant?

Reply:
+1, now taking it from the parameters passed